docs/containers/bridge-to-kubernetes.md

```diff
@@ -156,46 +156,56 @@ Save your changes and press `kb(workbench.action.debug.restart)` or click **Run*
 
 Click **Run** then **Stop Debugging** or press `kb(workbench.action.debug.stop)` to stop the debugger.
 
-> **Note**: By default, stopping the debugging task also disconnects your development computer from your Kubernetes cluster. You can change this behavior by searching for **Local Process With Kubernetes: Disconnect After Debugging** in the Visual Studio Code settings and removing the check next to **Disconnect automatically when Debugging ends**. After updating this setting, your development computer will remain connected when you stop and start debugging. To disconnect your development computer from your cluster click on the LPK extension on the status bar then choose **Disconnect current session**.
+> **Note**: By default, stopping the debugging task also disconnects your development computer from your Kubernetes cluster. You can change this behavior by searching for **Bridge to Kubernetes: Disconnect After Debugging** in the Visual Studio Code settings and removing the check next to **Disconnect automatically when Debugging ends**. After updating this setting, your development computer will remain connected when you stop and start debugging. To disconnect your development computer from your cluster click on the Bridge to Kubernetes extension on the status bar then choose **Disconnect current session**.
 >
-> If Visual Studio Code abruptly ends the connection to the cluster or terminates, the service you are redirecting may not be restored to its original state before you connected with Local Process with Kubernetes. To fix this issue, see the [Troubleshooting guide][troubleshooting].
+> If Visual Studio Code abruptly ends the connection to the cluster or terminates, the service you are redirecting may not be restored to its original state before you connected with Bridge to Kubernetes. To fix this issue, see the [Troubleshooting guide][troubleshooting].
```

Comment:
Could you please remove this last sentence? This bug of not restoring properly was fixed a couple of weeks ago. Also, the Troubleshooting page we're referring to here is about to get removed as part of Zach's PR.